content/books/deathworlders/chapter-40-war-two-worlds-pt1-instigation.md

```diff
@@ -2240,8 +2240,8 @@ Tick.
 
 “How?”
 
```

Comment:
The items below from the Eneity’s conversation deliberately have excess spaces to make them newline appropriately.